fix the issues in deploy-operator.md (#80)
observatorium/operator: docs/deploy-operator.md

#### S3 storage endpoint and secret

For **testing purposes** you may use [minio](https://github.com/minio/minio) as described below.

```shell
kubectl create namespace observatorium-minio
kubectl apply -f https://raw.githubusercontent.com/observatorium/observatorium/main/configuration/examples/dev/manifests/minio-secret-loki.yaml
kubectl apply -f https://raw.githubusercontent.com/observatorium/observatorium/main/configuration/examples/dev/manifests/minio-secret-thanos.yaml
kubectl apply -f https://raw.githubusercontent.com/observatorium/observatorium/main/configuration/examples/dev/manifests/minio-pvc.yaml
kubectl apply -f https://raw.githubusercontent.com/observatorium/observatorium/main/configuration/examples/dev/manifests/minio-deployment.yaml
kubectl apply -f https://raw.githubusercontent.com/observatorium/observatorium/main/configuration/examples/dev/manifests/minio-service.yaml
```

#### Dex

> For **testing purposes** you may use [dex](https://github.com/dexidp/dex) which is an identity service for observatorium API as described below.

Reviewer: small nit:
```suggestion
For **testing purposes** you may use [dex](https://github.com/dexidp/dex) as an OIDC identity provider for the Observatorium API as described below.
```

kubectl apply -n observatorium -f https://raw.githubusercontent.com/observatorium/operator/master/example/manifests/observatorium.yaml
```

> For **testing purposes**, you may use the certificate and key as described below.

Reviewer: ```suggestion
For **testing purposes**, you may use the end-to-end test certificate and key as described below.
```
